src/data/test-game.ts

```diff
@@ -53,127 +60,379 @@ export default {
       },
       {
-        challenge: "Spot a wooden shoe",
-        description: "Find a wooden shoe in the Netherlands",
-        points: 10,
+        challenge: "Capture a mountain peak",
+        description: "Photograph the summit of a prominent mountain",
+        points: 40,
         type: "normal",
+        image:
+          "https://www.ephotozine.com/resize/articles/15270/P9080276.jpg?RTUdGk5cXyJFCgsJVANtdxU+cVRdHxFYFw1Gewk0T1JYFEtzen5YdgthHHsvEVxR",
       },
       {
-        challenge: "Spot a windmill",
-        description: "Find a windmill in the Netherlands",
-        points: 25,
+        challenge: "Visit a botanical garden",
+        description:
+          "Explore and take photos of a botanical garden's diverse flora",
+        points: 20,
         type: "normal",
+        image:
+          "https://assets.cntraveller.in/photos/64199ada59df46f87e0a53ca/16:9/w_1024%2Cc_limit/GettyImages-470648565.jpg",
       },
       {
-        challenge: "Spot a tulip field",
-        description: "Find a field of tulips in the Netherlands",
-        points: 10,
+        challenge: "Find a local wildlife sanctuary",
+        description:
+          "Visit a wildlife sanctuary and capture the animals in their natural environment",
+        points: 45,
         type: "normal",
+        image:
+          "https://img.traveltriangle.com/blog/wp-content/uploads/2019/08/Wildlife-Sanctuaries-In-India.jpg",
       },
       {
-        challenge: "Spot a bike",
-        description: "Find a bike in the Netherlands",
-        points: 5,
+        challenge: "Discover a natural hot spring",
+        description: "Find and photograph a natural hot spring",
+        points: 40,
         type: "normal",
+        image:
+          "https://media.cntraveler.com/photos/5eb420d241269ff2d4f2ea8b/master/pass/Pamukkale-Denizli-Turkey-GettyImages-539479634.jpg",
       },
     ],
   ],
+  runs: {
+    "summer-edition": {
+      name: "Summer Edition",
+      date: "Aug 24",
+      lastUpdate: "13:01",
+      teams: {
+        "team-a": {
+          name: "Team A",
+          challenges: [
+            {
+              name: "Discover a natural hot spring",
+              date: "Aug 24",
+              comment: `_Great job_!!!  \n![Clapping](/images/clapping-1.gif)`,
+            },
+            {
+              name: "Spot a wooden shoe",
+              date: "Aug 24",
+              comment: `_Great job_!!!  \n![Clapping](/images/clapping-2.gif)`,
+            },
+            {
+              name: "Try a traditional dish",
+              date: "Aug 24",
+              comment: `_Great job_!!!  \n![Clapping](/images/clapping-3.gif)`,
+            },
+          ],
+        },
+        "team-b": {
+          name: "Team B",
+          challenges: [
+            {
+              name: "Capture a sunrise over the ocean",
+              date: "Aug 24",
```

Comment:
Perhaps we can assume this date and all the other dates to also be a UTC string? and inside the component, we convert the UTC string into a date object and then display in a standard format? 🤔

Reply:
https://github.com/geo-quest/travel-bingo/pull/19/commits/4804dfa7213beedfd92edb2f4eb55652119fb470 ;-)